requirements/base.in

```diff
@@ -5,5 +5,4 @@
 Django>=1.11              # Web application framework
 django-waffle             # Allows for feature toggles in Django.
 newrelic                  # New Relic agent for performance monitoring
-psutil==1.2.1             # Library for retrieving information on running processes and system utilization
-                          # NOTE: psutil pinned to match edx-platform version. Will need to be updated at the same time.
+psutil                    # Library for retrieving information on running processes and system utilization
```

Comment:
@jmbowman: Do you mind providing more details about this pin in edx-platform so we can either remove this, or maybe clarify the comment here and point to a ticket?  See https://github.com/edx/edx-platform/blame/master/requirements/edx/paver.in#L20 for your pin in edx-platform.

Reply:
I looked at that commit, and the pin had been carried over from previous pinning. I think it's just historical baggage, but I'm happy to be corrected.

Reply:
Yeah, I looked into unpinning it in edx-platform and saw that it would need code changes in at least one dependency (not sure if this is the only one).  Punted on it as it wasn't urgent at the time.